### PR TITLE
Issues pagination

### PR DESCRIPTION
**Pull request by danielmarquesr**
## Proposed Changes

Add pagination of issues.

As discussed earlier with maintainers @alaxalves @MatheusRich , the pagination feature was implemented using vuetify.

Solve: #249 

## Type of Changes

What type of change this Pull Request brings to Falko?
_Put an `x` in the boxes that apply_

- [ ] Bugfix
- [x] New Feature

## Checklist

_Put an `x` in the boxes that apply. If you're confused about any of the following topics, ask us. We're here to help you!_

- [x] This Pull Request has a significant name.
- [x] The build is okay (tests, code climate).
- [x] This Pull Request mentions a related issue.

## Screenshots
falko-page

## Further Comments
* This new feature must be accepted along with the backend modification. ([Pull Request - Back end](https://github.com/falko-org/Falko-API/pull/246))
* I noticed that in the App.vue file there is something wrong with the height of the part of the page content. As a result, it is not possible to place the paging component at the bottom of the page.

### CODE REVIEW

**Review by alaxalves — APPROVED** [review]
Look pretty solid to me!! Thank you so much @danielmarques28. What do you think @MatheusRich ??

**Comment by MatheusRich** [comment]
@danielmarques28 Looks good! I'll merge this after the backend is merged too. After this, I think we should list issues in a List rather than this matrix style.

**Pull request merged**
